### PR TITLE
chore: add support for driver cli args

### PR DESCRIPTION
**Pull request by ayvnkhan**

### CODE REVIEW

File: lib/basedriver/driver.js [hunk 1]
```diff
@@ -48,6 +48,9 @@ class BaseDriver extends Protocol {
     this.originalCaps = null; // To give the original capabilities to reset
     this.helpers = helpers;
 
+    //allow for cli args to be used by drivers
```
Comment (jlipps) [inline lib/basedriver/driver.js:51]: nit: `// ` (with space after)

**Review by jlipps — APPROVED** [review]
this looks good. note that because of the monorepo work that @boneskull is doing, anything going into the 2.0 branch on this repo may need to go into appium/appium instead

File: lib/basedriver/driver.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ class BaseDriver extends Protocol {
    */
   static baseVersion = BASEDRIVER_VER;
 
-  constructor (opts = {}, shouldValidateCaps = true) {
+  constructor (opts = {}, driverArgs = {}, shouldValidateCaps = true) {
```
Comment (mykola-mokhnach) [inline lib/basedriver/driver.js:41]: this might break existing drivers. Consider adding a new argument to the end rather than in the middle

**Review by mykola-mokhnach — CHANGES_REQUESTED** [review]
a potential breaking change is present

**Review by mykola-mokhnach — APPROVED** [review]

**Pull request closed without merging**